packages/react/src/exampleTypes/StepSlider.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+import { act, render, screen } from '@testing-library/react'
+
+import StepSlider from './StepSlider'
+
+import testTree from './stories/testTree'
+import { renderWithProviders } from '../utils/test'
+
+describe('StepSlider', () => {
+  beforeEach(() => {
+    renderWithProviders(<StepSlider tree={testTree()} path={['step']} />)
+  })
+
+  it('renders range slider', async () => {
+    const rangeSlider = await screen.getByTestId('chakra-range-slider-track')
+    const rangeSliderThumbs = await screen.getAllByRole('slider')
```

Comment:
💎 Minor _potential optimization_ which reuses the query above. From there, I _wonder_ if we'd need to use `await` for rangeSliderThumbs?
```suggestion
    const rangeSliderThumbs = await within(rangeSlider).getAllByRole('slider')
```

Reply:
Veeery nice! Almost forgot about this. However, unfortunately this approach doesn't apply here as `chakra-range-slider-track` and the two sliders are siblings:
```
<div id="slider-root-5" tabindex="-1" class="chakra-slider css-1kveibn">
  <div
    id="slider-track-5"
    class="chakra-slider__track css-ucasgs"
    data-testid="chakra-range-slider-track"
  >
    <div
      id="slider-filled-track-5"
      class="chakra-slider__filled-track css-1uaoubu"
    ></div>
  </div>
  <div role="slider" tabindex="0"></div>
  <div role="slider" tabindex="0"></div>
</div>

```

Reply:
Ahhh haha my bad, I was imagining the sliders as children. Thanks!